commands/ft.search.md

```diff
@@ -0,0 +1,235 @@
+```
+FT.SEARCH <index> <query>
+  [NOCONTENT]
+  [TIMEOUT <timeout>]
+  [PARAMS nargs <name> <value> [ <name> <value> ...]]
+  [LIMIT <offset> <num>]
+  [DIALECT <dialect>]
+```
+
+Performs a search of the specified index. The keys which match the query expression are returned.
+
+- **\<index\>** (required): This index name you want to query.
+- **\<query\>** (required): The query string, see below for details.
+- **NOCONTENT** (optional): When present, only the resulting key names are returned, no key values are included.
+- **TIMEOUT \<timeout\>** (optional): Lets you set a timeout value for the search command. This must be an integer in milliSeconds.
+- **PARAMS \<count\> \<name1\> \<value1\> \<name2\> \<value2\> ...** (optional): `count` is of the number of arguments,
+    i.e., twice the number of value name pairs. See the query string for usage details.
+- **RETURN \<count\> \<field1\> \<field2\> ...** (options): `count` is the number of fields to return.
+    Specifies the fields you want to retrieve from your documents, along with any aliases for the returned values.
+    By default, all fields are returned unless the NOCONTENT option is set, in which case no fields are returned.
+    If num is set to 0, it behaves the same as NOCONTENT.
+- **LIMIT \<offset\> \<count\>** (optional): Lets you choose a portion of the result.
+    The first `<offset>` keys are skipped and only a maximum of `<count>` keys are included. The default is LIMIT 0 10,
+    which returns at most 10 keys.
+- **DIALECT \<dialect\>** (optional): Specifies your dialect. The only supported dialect is 2\.
+
+**RESPONSE**
+
+The command returns either an array if successful or an error.
```

Comment:
This will get printed out twice, once from the replies section that is generated and here. I think we can omit most of this still.